config/config.go

```diff
@@ -43,6 +44,10 @@ type (
 		HttpServer HttpServerConfig `yaml:"httpServer"`
 		// Rpc is the config for rpc calls
 		Rpc RpcConfig `yaml:"rpc"`
+		// AsyncAddresses are the addresses for API service to call AsyncServices' internal APIs
+		AsyncAddresses []string `yaml:"asyncAddresses"`
+		// the total shard count in AsyncServices. default to be 1.
+		AsyncShard int `yaml:"asyncShard"`
```

Comment:
It's better to use the same value below to avoid mistakes(what if they are adifferent) and I would prefer to put them together

Reply:
I could not get it. Which same value are you talking about?

Reply:
Moved to database config.